third_party/Makefile

```diff
@@ -5,11 +5,16 @@
 
 SB_MICRO_PATH ?= "/usr/local"
 MPI_HOME_PATH ?= "/usr/local/mpi"
+HIP_HOME ?= "/opt/rocm/hip"
+RCCL_HOME ?= "/opt/rocm/rccl"
 
-.PHONY: all cutlass bandwidthTest nccl_tests perftest fio
+.PHONY: all common cuda rocm cutlass bandwidthTest nccl_tests perftest fio rccl_tests
 
 # Build all targets.
-all: cutlass bandwidthTest nccl_tests perftest fio
+all: cuda rocm
```

Comment:
wouldn't it better to add common to this line so when someone runs only rocm or cuda, does not run fio?

Reply:
For rocm or cuda, we would like to call `make cuda` for running SuperBench in CUDA and call `make rocm` for running SuperBench in ROCM. We think there is very limited case to run `make all`. In future, we may need to add `make cpu`.

Reply:
Does running `make cuda` or `make rocm` in this case, run fio as well? If yes, is it expected?

Reply:
Yes, it will install the FIO because make cuda would be running on the CUDA docker. Currently, if we have a cuda docker and we run `make all`, it will be crash. That's why we design `make cuda` and `make rocm`.